### PR TITLE
v2 Data & Importers

### PR DESCRIPTION
**Pull request by ghickman**
This adds the v2 data CSVs, importer functions which can import them, and updates the DB models to handle some changes this data has introduced.

I've restructured the v1 importers so a single management command can load all the data from a given version, defaulting to the latest one.

While the v2 data has introduced some schema changes I don't expect this to happen with each new version going forwards.  Thus the structure of the importers and data was picked with the expectation that there will a) be future versions added and b) previous versions should still be accessible.

### CODE REVIEW

**Comment by fredkingham** [comment]
@ghickman its conflicted

**Comment by ghickman** [comment]
@fredkingham – fixed!

File: csv_schema/models.py [hunk 1]
```diff
@@ -33,24 +25,23 @@ def turn_preview_mode_off(sender, user, request, **kwargs):
...
 
 class Database(BaseModel):
     SEARCH_FIELDS = [
         "display_name", "name", "description", "link"
     ]
     name = models.CharField(max_length=255, unique=True)
-    display_name = models.CharField(
-        max_length=255, unique=True, blank=True, null=True
-    )
+    display_name = models.TextField(blank=True, null=True)
```
Comment (fredkingham) [inline csv_schema/models.py:40]: Why are we removing the uniqueness of db display name? Surely this is something we want?

**Comment by fredkingham** [comment]
When running migrations on an existing db, I get 
```
NOT NULL constraint failed: csv_schema_table.schema_id
```

which is expected. Can we make sure that Rob hasn't actually changed any of the data on the test server and that's it ok to nuke? (He asked whether he could update that data a month ago, so just want to make sure, it is backed up to S3 but still...)

**Comment by fredkingham** [comment]
The new import command is `./manage.py load_data` right? Can we update the ReadMe? (I'd do it myself, but I wanted to check I'm right!)

**Comment by fredkingham** [comment]
After running load_data I have no databases or anything 😕

**Comment by fredkingham** [comment]
also `python ./manage.py initial_load` doesn't work anymore

File: csv_schema/models.py (hunk 1 above)
Comment (ghickman) [reply csv_schema/models.py:40]: Lines [43](https://github.com/nhsengland/NCDR-reference-library/pull/51/files#diff-1a7ed64766867c4252cb60307a8d2d1aR43) and [65](https://github.com/nhsengland/NCDR-reference-library/pull/51/files#diff-1a7ed64766867c4252cb60307a8d2d1aR65) have the same `Name` value for different database names (`NHSE_Mental_Health` and `NHSE_MH_V3_YTD` respectively).

Do you think this is something we should push back on?

**Comment by ghickman** [comment]
Readme updated in 7b3a83f

I removed `initial_load` in favour of `load_data`.

> I have no databases or anything

Is this on the website or in your local database?  If it's the website that's because all databases will be unpublished after importing.  Should loading data publish it?

**Comment by fredkingham** [comment]
So http://127.0.0.1:8000/database/NHSE_SUSPlus_Reporting/AE/ is blowing up

**Pull request merged**